src/lib/ptree/ptree.lua

```diff
@@ -347,10 +347,11 @@ function Manager:monitor_worker_counters(id)
    local events = inotify.recursive_directory_inventory_events(dir, cancel)
    for ev in events.get, events do
       if has_suffix(ev.name, '.counter') then
+         local islink = S.lstat(ev.name).islnk
```

Comment:
Given that this event handler runs asynchronously, I think we may need some error handling around the S.lstat call.  WDYT?

Reply:
Good catch, you’re right! I think I even remember seeing errors coming from this and not connecting the dots.

Reply:
Should be fixed in 20c303c